samconfig.toml

```diff
@@ -0,0 +1,10 @@
+version = 0.1
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "aws-batch-metrics"
+s3_bucket = "aws-sam-cli-managed-default-samclisourcebucket-15sh4i0bj0yfx"
+s3_prefix = "aws-batch-metrics"
+region = "eu-west-1"
```

Comment:
Is there a way to not hardcode the region, and instead have SAM use the AWS account's default region?